src/app/views/input/report.py

```diff
@@ -1,17 +1,18 @@
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
+from typing import Optional
 from pydantic.fields import Field
 
 
 class Equipment(BaseModel):
-    equip_head_id: int = Field(None, ge=0)
-    equip_amulet_id: int = Field(None, ge=0)
-    equip_torso_id: int = Field(None, ge=0)
-    equip_legs_id: int = Field(None, ge=0)
-    equip_boots_id: int = Field(None, ge=0)
-    equip_cape_id: int = Field(None, ge=0)
-    equip_hands_id: int = Field(None, ge=0)
-    equip_weapon_id: int = Field(None, ge=0)
-    equip_shield_id: int = Field(None, ge=0)
+    equip_head_id: Optional[int] = Field(0, ge=0)

```

Comment:
Any reason why field is not default None

Reply:
if we set 0 here then on the db side we will get 0 instead of none if the key is not provided